shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -2654,8 +2654,10 @@ - (void)testFlutterClippingMaskViewPoolReuseViewsAfterRecycle {
   CGRect newRect = CGRectMake(0, 0, 10, 10);
   FlutterClippingMaskView* view3 = [pool getMaskViewWithFrame:newRect];
   FlutterClippingMaskView* view4 = [pool getMaskViewWithFrame:newRect];
-  XCTAssertEqual(view1, view3);
-  XCTAssertEqual(view2, view4);
+  // view3 and view4 should randomly get either of view1 and view2.
+  XCTAssertTrue((view1 == view3) || (view1 == view4));
+  XCTAssertTrue((view2 == view3) || (view2 == view4));
+  XCTAssertNotEqual(view3, view4);
```

Comment:
nit: can you do 

`assert equal(new set of v1 & v2, new set of v3 & v4)`

Reply:
+1